components/utils/Button/index.jsx

```diff
@@ -1,6 +1,6 @@
-export default function Button({ text, type, bg_color, fg_color, border, padding, onClick}) {
+export default function Button({ text, type, onClick, style}) {
```

Comment:
```suggestion
export default function Button({ text, type, onClick, customStyle}) {
```

I think we should change the name of this custom style property because, style is a html attribute for inline styles.